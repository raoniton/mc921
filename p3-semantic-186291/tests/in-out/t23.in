class Main {
    public int f(int x, int x) {
        return x+y;
    }
    public static void main(String[] args) {
        this.f(1, 2);
    }
}