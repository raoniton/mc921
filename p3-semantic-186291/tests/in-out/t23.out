SemanticError: Parameter x has already been defined @ 2:25
